wix/yoshi
# [WiP] Use webpack-dev-server for CDN

### 🔦 Summary
Attempt at #753 

This PR:
* Uses code from https://github.com/wix/yoshi/pull/752 to make the tests stable.
* Modifies the tests that are testing `"hmr": true` (of `webpack-hot-client`) to test `/webpack/hot/` instead.
* Moves code from `start-app` to `webpack-util` to be later used in both the CDN and start-app
* Removes `serve-handler` dependency in favour of `webpack-dev-server` `contentBase`

![screen shot 2018-11-14 at 15 26 59](https://user-images.githubusercontent.com/326402/48497310-8ca36380-e83c-11e8-9b08-c38722a71846.png)



### 🗺️ Test Plan
* Verify all tests are passing.
* Use with existing project (I've tested with `form-builder` and client only project with `create-yoshi-app`)
